src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/ORMQueryBuilderLoaderTest.php

```diff
@@ -17,25 +17,6 @@
 
 class ORMQueryBuilderLoaderTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * @expectedException \Symfony\Component\Form\Exception\UnexpectedTypeException
-     */
-    public function testItOnlyWorksWithQueryBuilderOrClosure()
-    {
-        new ORMQueryBuilderLoader(new \stdClass());
```

Comment:
ORMQueryBuilderLoader is going to be type hinted in 3.0 so this test is legacy also